onnx/onnx-operators_pb.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
```

Comment:
This file also needs [these boilerplates](https://github.com/onnx/onnx/blob/master/onnx/onnx_pb.h#L11).

Reply:
I'm adding #include "onnx/onnx_pb.h" in this file.